jobs/queue-processor/src/main.py

```diff
@@ -108,8 +107,11 @@ async def get_next_queue_item():
             for record in records:
                 logger.info(f"Found {record['count']} item(s) for {record['version']}.")
                 version = json.loads(record["version"])
-                # currently set one worker per 5 queue items
-                num_of_workers = math.ceil(record["count"] / MAX_JOBS_PER_WORKER)
+                # spin up max 5 workers, one per first five jobs in queue
+                num_of_workers = (
```

Comment:
could do `num_of_workers = min(record["count"], MAX_NUM_OF_WORKERS)`

Reply:
Yeah I like that a lot